datapipelines/src/test/java/io/customer/datapipelines/sdk/CustomerIOBuilderTest.kt

```diff
@@ -0,0 +1,135 @@
+package io.customer.datapipelines.sdk
+
+import android.app.Activity
+import android.app.Application
+import io.customer.commontest.BaseUnitTest
+import io.customer.commontest.module.CustomerIOGenericModule
+import io.customer.datapipelines.DataPipelinesModule
+import io.customer.datapipelines.extensions.apiHost
+import io.customer.datapipelines.extensions.cdnHost
+import io.customer.sdk.CustomerIOBuilder
+import io.customer.sdk.core.di.SDKComponent
+import io.customer.sdk.core.util.CioLogLevel
+import io.customer.sdk.data.model.Region
+import io.customer.sdk.extensions.random
+import org.amshove.kluent.shouldBe
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.never
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class CustomerIOBuilderTest : BaseUnitTest() {
+
+    private val application: Application = Robolectric.buildActivity(Activity::class.java).setup().get().application
```

Comment:
I think this should be moved to some base class as we'll keep needing this for several tests. I'm okay to not changes it in this PR, we can cover this in Setup Testing Environment ticket. But adding note so we don't miss this.

Reply:
This is robolectric specific, not sure if its wise to make our base test robolectric, either we create another robolectric base class that extends base class or let each class add this if needed.

Reply:
Yes, I meant to have another base class to help similar cases, we can look into this later with test changes though, no action required for this PR.